### PR TITLE
Product page updates for cdk, devstudio and devsuite

### PR DESCRIPTION
**Pull request by Dantheman720**

### CODE REVIEW

**Comment by Dantheman720** [comment]
@MikGue - This is a work in progress for the Product page updates that were requested

**Comment by LightGuard** [comment]
@Dantheman720 merge conflict, please fix

File: products/devsuite/_common/product.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 name: Red Hat Development Suite
-abbreviated_name: DevSuite
+abbreviated_name: Red Hat Development Suite
 product_logo: enterprise-app-platform.png
-current_version: 7.0.0
+current_version: 1.1.0
```
Comment (LightGuard) [inline products/devsuite/_common/product.yml]: We went from 7.0.0 to 1.1.0?

File: products/devsuite/docs-and-apis.adoc [hunk 2]
```diff
@@ -0,0 +1,3 @@
+:awestruct-layout: product-docs-and-apis
+:awestruct-interpolate: true
```
Comment (LightGuard) [inline products/devsuite/docs-and-apis.adoc]: Maybe there's more coming, but if there's no content, do we need the interpolate?

Comment (Dantheman720) [inline products/devsuite/docs-and-apis.adoc]: I don't think we'll need to interpolate. I'll remove that.

File: products/devsuite/_common/product.yml (hunk 1 above)
Comment (Dantheman720) [inline products/devsuite/_common/product.yml]: I changed that because they were discussing 1.1. I was wondering why this was listed as 7.0

**Comment by rctrh** [comment]
Sorry, I sent another PR to @Dantheman720's repo to be part of this PR.  I missed that the change in filenaming from 1.0 (developer-suite-1.x) to 1.1 (devsuite-1.x).   Please pick up those two commits before merging.

**Comment by rctrh** [comment]
@Dantheman720 One more change to URLs coming in 282dfe8.

**Pull request merged**